markhor_bringup/launch/cameras/markhor_video0_cam.launch

```diff
@@ -1,5 +1,5 @@
 <launch>
-    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" ns="markhor/video0" output="screen">
+    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" ns="markhor/camera_00" output="screen">
```

Comment:
So in `markhor_video0_cam` we are launching `markhor/camera_00/usb_cam`? Would you mind maybe changing the name of the file to `camera_00`? Also, why 2 digits, I doubt will have more than 10 cameras.

Reply:
`camera_0X` naming is to match the naming convention of the `ros_astra_camera` nodes. They were used to develop multipliers, so it was easier to keep matching topic names than to ask our colleagues to adjust their respective codes.
I agree for the file names, I will change the four file names and their usage locations later today if I can, tomorrow at the latest.